apps/hmc/hmc-judicial-payment-service-int/hmc-judicial-payment-service-int.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
```

Comment:
better to add just one app at at time rather than multiple (the base + the int you've done)

less to review and less to change if any issues

Reply:
Removed int